CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ### 0.1.2 (Next)
 
-* Your contribution here.
+### 0.1.2 (8/18/2022)
```

Comment:
Still incorrect :) Just one entry for 0.1.2 (Next)
Leave * Your contribution here in there.

Sorry to be a hassle.

Reply:
Fixed.

Reply:
Not fixed, but I'll merge and then make a PR so we can see danger is working.